.github/workflows/create-release.yaml

```diff
@@ -67,17 +75,45 @@ jobs:
           RELEASE_ID=$(./scripts/create_draft_release.sh $RELEASE_VERSION)
           echo "release_id=$RELEASE_ID" >> $GITHUB_OUTPUT
 
-      - name: Add lightweight tag to trigger release EPP build job
+  wait-for-build-job:
+    name: Wait for prow jobs
+    needs: [ verify-release, create-draft ]
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout eventing-tools
+        uses: actions/checkout@v4
+        with:
+          repository: 'kyma-project/eventing-tools'
+          path: 'kyma-project/eventing-tools'
+          ref: main
+          sparse-checkout: 'scripts/wait-for-commit-check'
+
+      - name: Setup python
+        uses: actions/setup-python@v5
+        with:
+          python-version: '3.10'
```

Comment:
Is Python 3.10 fine? Can we use 3.11 or 3.12?

Reply:
yes, its used everywhere with Python 3.10. Technically, it will work with 3.11 or 3.12 but we can do the upgrade later at all places where is it used.

Reply:
Ok. Thanks :-)